### PR TITLE
Add keyboard shortcuts for 'run' and 'stop'

### LINKED ISSUE #16: ⌘T for reformat and ⌘R map to run
Thanks!
👍

### PR DESCRIPTION
**Pull request by kaganjd**
- Fixes #16.

### CODE REVIEW

File: client/modules/IDE/pages/IDEView.js [hunk 1]
```diff
@@ -123,6 +123,16 @@ class IDEView extends React.Component {
       e.preventDefault();
       e.stopPropagation();
       this.props.saveProject();
+    } else if (e.key === 'Enter' && e.shiftKey && ((e.metaKey && this.isMac) || (e.ctrlKey && !this.isMac))) {
+      console.log('stopping sketch');
```
Comment (catarak) [inline client/modules/IDE/pages/IDEView.js]: Everything looks good, just want to remove the console.logs

File: client/modules/IDE/pages/IDEView.js [hunk 2]
```diff
@@ -123,6 +123,16 @@ class IDEView extends React.Component {
...
+      e.preventDefault();
+      e.stopPropagation();
+      this.props.stopSketch();
+    } else if (e.key === 'Enter' && ((e.metaKey && this.isMac) || (e.ctrlKey && !this.isMac))) {
+      console.log('starting sketch');
```
Comment (catarak) [inline client/modules/IDE/pages/IDEView.js]: Remove this console.log as well

File: client/modules/IDE/pages/IDEView.js (hunk 1 above)
Comment (kaganjd) [inline client/modules/IDE/pages/IDEView.js]: done

File: client/modules/IDE/pages/IDEView.js (hunk 2 above)
Comment (kaganjd) [inline client/modules/IDE/pages/IDEView.js]: done

**Pull request merged**